# (svelte/--)

> description

## :book: Rule Details

This rule reports ???.

<!--eslint-skip-->

```svelte
<script>
  /* eslint svelte/--: "error" */
</script>

<!-- ✓ GOOD -->

<!-- ✗ BAD -->
```

## :wrench: Options

```json
{
  "svelte/--": ["error", {}]
}
```

-

## :books: Further Reading

-
